android/canonical/app/src/main/java/com/google/samples/quickstart/canonical/MapsFragment.kt

```diff
@@ -2,30 +2,49 @@ package com.google.samples.quickstart.canonical
 
 import android.Manifest
 import android.app.Activity
+import android.content.ContentValues.TAG
```

Comment:
This TAG is usually not something you import, rather a string constant you define locally. In this case maybe add private const val TAG = "MapFragment" in the companion object.